app/src/main/java/com/amaze/filemanager/activities/PreferencesActivity.java

```diff
@@ -144,12 +144,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
-    public void setChanged() {
-        changed = true;
+    public void setRestartActivity() {
+        restartActivity = true;
     }
 
-    public boolean getChanged() {
-        return changed;
+    public boolean getRestartActivity() {
```

Comment:
Not sure if others would agree... but to my knowledge for boolean getters, it would be better to say `isRestartActivity()` here.